pkg/vault/contracts/CompositeLiquidityRouter.sol

```diff
@@ -347,15 +384,20 @@ contract CompositeLiquidityRouter is ICompositeLiquidityRouter, BatchRouterCommo
             IERC4626 wrappedToken = IERC4626(address(erc4626PoolTokens[i]));
             IERC20 underlyingToken = IERC20(_vault.getBufferAsset(wrappedToken));
 
+            if (useWrappedTokens[i] == true) {
+                finalAmountsIn[i] = amountsIn[i];
+                _takeTokenIn(params.sender, wrappedToken, finalAmountsIn[i], params.wethIsEth);
+                continue;
```

Comment:
Same as above; we need to apply limits here as in [here](https://github.com/balancer/balancer-v3-monorepo/pull/1201/files#diff-06fda6cf7b962215183d91acc7ad1adf10fc3260b79cd46e9a0c70a78526c88fR397)